### ISSUE TITLE
[1.10] Composer is missing composer-runtime-api v1 package

### ISSUE THREAD

**Issue by Jean85** [composer/composer#8846]
## Main issue
Composer 1 is missing the meta-package `composer-runtime-api` and its proper version, in the same way as it has `composer-plugin-api`. This causes an incomplete error message with `composer require`, and a false negative with `composer proibits` / `composer why-not`

## Reproduction
When I run this command with **Composer 1** in an empty folder:
```
composer require jean85/pretty-package-versions:^1.2
composer require jean85/pretty-package-versions:^1.4
composer why-not jean85/pretty-package-versions:^1.4
```

I get the following output:
 * First command goes ok
 * second command errors out with:
```
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for jean85/pretty-package-versions ^1.4 -> satisfiable by jean85/pretty-package-versions[1.4.0].
    - jean85/pretty-package-versions 1.4.0 requires composer-runtime-api ^2.0 -> no matching package found.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://getcomposer.org/doc/04-schema.md#minimum-stability> for more details.
 - It's a private package and you forgot to add a custom repository to find it

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.

Installation failed, reverting ./composer.json to its original content.
```
Third command does NOT report any problem:
```
There is no installed package depending on "jean85/pretty-package-versions" in versions not matching ^1.4
```

And I expected this to happen:
 * second command should fail due to conflict with `composer-runtime-api` v1
 * third command should say the same

**Comment by Seldaek** [comment]
Thanks for the fix, I'll try and release a new 1.10 patch release with this soon.

**Comment by Seldaek** [comment]
1.10.6 is out fixing this
